wwwroot/404.html

```diff
@@ -1,95 +1,50 @@
-    <div class="row">
-        <div class="col-md-2">
-            <img height="105" class="logo center-block" src="/public/img/pmc-stacked.jpg" style="opacity:0.6" alt="Logo of Department of the Prime Minister and Cabinet"><!-- match desaturated GA logo. -->
-            <img height="80" class="logo center-block" src="/public/img/GA-dark.png" alt="Logo of Geoscience Australia">
-            <img height="80" class="logo center-block" src="/public/img/data61-white.jpg" alt="Logo of Data61/CSIRO">
-            <img height="140" class="logo center-block" src="/public/img/iawards.png" alt="Logo of iAwards">
-            <img width="160" class="logo center-block" src="/public/img/Finalist-Innovation.gif" alt="Finalist in Innovation at CeBIT AU Business Awards 2015">
-        </div>
-        <div class="col-md-9">
-
-        <h2>That page doesn't seem to be here</h2>
-        <p>The page you're looking for doesn't seem to exist. If you think this is an error, please send an email to <a href="mailto:data@pmc.gov.au">data@pmc.gov.au</a>.</p>
-        <p><b>&larr; Return to <a href="/">NationalMap</a></b>.
-        <div style="font-size: 300px; color: lightgray; text-align:center">404</div>
-
-
-    </div>
-    </div>
-
-
-        </div>
-        </div>
-        <!-- Footer -->
-<footer class="footer">
-    <div class="container">
-        <div class="col-lg-12">
-            <p>&copy; Department of the Prime Minister and Cabinet, Department of Communications, and National ICT Australia Limited (NICTA) 2014-2015. All Rights Reserved.</p>
-            <p>
-                <strong><a href="#terms">Terms and conditions</a></strong> –
-                <strong><a href="privacy.html">Privacy</a></strong>
-            </p>
-        </div>
+  <head>
+    <meta charset="utf-8" />
+    <title>TerriaJS</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link
+      href="//fonts.googleapis.com/css?family=Roboto:300,600"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <style>
+      * {
+        box-sizing: border-box;
+      }
+      body {
+        font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+          "Helvetica Neue", Arial, sans-serif;
+        height: 100vh;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        gap: 1rem;
+        color: #111827;
+      }
+      .error-number {
+        font-family: inherit;
+        font-size: 2.5rem;
+        font-weight: 600;
+        line-height: 1.2;
+        margin: 0;
+        padding-right: 1rem;
+        border-right: 1px solid #dee2e6;
+      }
+      .error-description {
+        font-family: inherit;
+        font-size: 1.25rem;
+        font-weight: 400;
+        line-height: 1.2;
+        margin: 0;
+      }
+    </style>
+  </head>
+  <body>
+    <h1 class="error-number">404</h1>
+    <div class="inline-block align-middle">
+      <h2 class="error-description" id="desc">Not found.</h2>
```

Comment:
Same comment about no period in heading  and providing a clickable link to somewhere reasonable as for the 500 page.